zkevm-circuits/src/evm_circuit/execution/codecopy.rs

```diff
@@ -212,15 +212,19 @@ mod tests {
     use bus_mapping::evm::OpcodeId;
     use eth_types::{bytecode, evm_types::gas_utils::memory_copier_gas_cost, Word};
     use halo2_proofs::arithmetic::BaseExt;
+    use mock::TestContext;
     use num::Zero;
     use pairing::bn256::Fr;
 
-    use crate::evm_circuit::{
-        execution::copy_code_to_memory::test::make_copy_code_steps,
-        step::ExecutionState,
-        table::RwTableTag,
-        test::run_test_circuit_incomplete_fixed_table,
-        witness::{Block, Bytecode, Call, CodeSource, ExecStep, Rw, RwMap, Transaction},
+    use crate::{
+        evm_circuit::{
+            execution::copy_code_to_memory::test::make_copy_code_steps,
+            step::ExecutionState,
+            table::RwTableTag,
+            test::run_test_circuit_incomplete_fixed_table,
+            witness::{Block, Bytecode, Call, CodeSource, ExecStep, Rw, RwMap, Transaction},
+        },
+        test_util::run_test_circuits,
     };
 
     fn test_ok(src_addr: u64, dst_addr: u64, size: usize) {
```

Comment:
With `test_busmapping`, we should remove this? Or this is kept to be used for negative case testing in the future?

Reply:
This can now be remembered 👍